aws_build/build.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
+set -x
+
+GRAFTNODED_DEB_BUILD_DIR="/home/ubuntu/graftnoded"
+SUPERGRAFT_DEB_BUILD_DIR="/home/ubuntu/supergraft"
+USERNAME="graft"
+
+mkcd(){
+  mkdir $1 && cd $1
+}
+
+make_build(){
+cd /usr/src/
+git clone --recursive https://github.com/graft-project/graft-ng.git
```

Comment:
seems like root privileges only required to do this - please move source directory e.g. as a sub-directory of the BUILD_TOP